pkg/ctl/namespace/grant_permission_test.go

```diff
@@ -23,20 +23,6 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func TestGrantPermissionsCmd(t *testing.T) {
```

Comment:
any idea why do we remove these tests?

Reply:
The permission behavior changed by the pulsar. https://github.com/apache/pulsar/pull/7074
I will change the test behavior later.